install e-cal-popup.h includes

From: Ondrej Jirman <megous@megous.com>

calendar/ChangeLog:

2008-04-06  Ondrej Jirman  <megous@megous.com>

	** Fix for bug #525234

	* gui/Makefile.am:
	* gui/dialogs/Makefile.am:
	Install more include files that are useful to the out of tree evolution
	plugins.
---

 calendar/gui/Makefile.am         |    4 +++-
 calendar/gui/dialogs/Makefile.am |    6 ++++++
 2 files changed, 9 insertions(+), 1 deletions(-)


diff --git a/calendar/gui/Makefile.am b/calendar/gui/Makefile.am
index f45b9b6..2b7cfe2 100644
--- a/calendar/gui/Makefile.am
+++ b/calendar/gui/Makefile.am
@@ -49,7 +49,9 @@ component_LTLIBRARIES = libevolution-calendar.la
 ecalendarincludedir = $(privincludedir)/calendar/gui
 
 ecalendarinclude_HEADERS =	\
-	e-cal-config.h
+	e-cal-config.h		\
+	e-cal-popup.h		\
+	itip-utils.h
 
 search_files = tasktypes.xml memotypes.xml caltypes.xml
 
diff --git a/calendar/gui/dialogs/Makefile.am b/calendar/gui/dialogs/Makefile.am
index 2ade11e..6dd8a4c 100644
--- a/calendar/gui/dialogs/Makefile.am
+++ b/calendar/gui/dialogs/Makefile.am
@@ -13,6 +13,12 @@ INCLUDES = 								\
 	-DPREFIX=\""$(prefix)"\"					\
 	$(EVOLUTION_CALENDAR_CFLAGS)
 
+ecalendarincludedir = $(privincludedir)/calendar/gui/dialogs
+
+ecalendarinclude_HEADERS =	\
+	comp-editor-page.h	\
+	comp-editor.h
+
 noinst_LTLIBRARIES = libcal-dialogs.la
 
 libcal_dialogs_la_LIBADD =					\
